secure/storage/src/crypto_key_storage.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{error::Error, policy::Policy, storage::Storage, Value};
+use libra_crypto::ed25519::{compat, Ed25519PublicKey};
```

Comment:
compat should be destroyed, we should only use generate_for_testing ... the naming needs to be improved to perhaps just generate.

Reply:
Done, thanks! (Although yes, the generate_for_testing() method name is not ideal :( )